panther_analysis_tool/main.py

```diff
@@ -63,13 +64,15 @@
 from panther_analysis_tool.data_model import DataModel
 from panther_analysis_tool.destination import FakeDestination
 from panther_analysis_tool.enriched_event import PantherEvent
+from panther_analysis_tool.lookup_tables import lookup_table
```

Comment:
Reminder to include the package in `setup.py` (see [here](https://github.com/panther-labs/panther_analysis_tool/blob/199c2d294e25b9ef332045996544974ab970ed87/setup.py#L24)) so that it is included in the PyPI release.